packages/wallets/src/wallets/stellar.ts

```diff
@@ -65,7 +65,7 @@ export class StellarWallet extends Wallet<StellarChain> {
                     type: "contract-call",
                     contractId,
                     method,
-                    memo,
+                    memo: memo != null ? { type: "text", value: memo } : undefined,
```

Comment:
@alberto-crossmint does this make sense? we were passing a string but the input now requires an object. only handling the `text` type